### PR TITLE
WRP-3074: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated package dependencies to fix security vulnerabilities.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3074

### CODE REVIEW

File: moonstone/pattern-analytics-webostv/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,473 @@
 {
   "name": "pattern-analytics-webostv",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (daniel-stoian-lgp) [inline moonstone/pattern-analytics-webostv/package-lock.json:4]: please remove this new added file from the PR

**Review by daniel-stoian-lgp — COMMENTED** [review]
just one comment

**Review by alexandrumorariu — COMMENTED** [review]
Please update this dependencies also: 
Samples => moonstone => pattern-dynamic-panel: @enact/moonstone  ^4.5.1  →  ^4.5.2
Samples => moonstone => pattern-single-panel-redux: @enact/moonstone  ^4.5.1  →  ^4.5.2

File: moonstone/pattern-analytics-webostv/package-lock.json (hunk 1 above)
Comment (adrian-cocoara-lgp) [reply moonstone/pattern-analytics-webostv/package-lock.json:4]: This file already exists on develop branch
image

maybe we should keep it?

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**